src/test/groovy/pl/tomaszdziurko/jvm_bloggers/mailing/BlogPostForMailItemSpec.groovy

```diff
@@ -26,13 +32,31 @@ class BlogPostForMailItemSpec extends Specification {
             Blog author = stubAuthorWith(name, twitter)
             BlogPost post = Stub(BlogPost) {
                 getBlog() >> author
+                getUrl() >> url
+                getTitle() >> title
             }
         when:
             BlogPostForMailItem blogPostForMailItem = BlogPostForMailItem.builder().from(post).build()
         then:
             blogPostForMailItem.authorLabel == "<a href=\"https://twitter.com/" + twitter.substring(1) + "\">" + name + "</a>"
     }
 
+    def "Should store author name as a label if twitter account is set with empty or null string"() {
```

Comment:
The test case formatting does not follow [contribution guide](https://github.com/tdziurko/jvm-bloggers/blob/master/CONTRIBUTING.md#testing).